resolwe_bio/processes/clustering/hierarchical_clustering.yml

```diff
@@ -56,3 +56,65 @@
     bash: |
       genehcluster.py {% for e in etcs %}"{{e.etcfile.file}}" {% endfor %}-e {% for e in etcs %}"{{e|id}}" {% endfor %}{% if genes %}-g {{genes|join:" "}}{% endif %} -l {{linkage}} -d {{distance}}
       echo "{\"proc.progress\":1.0,\"proc.rc\":$?}"
+
+
+- slug: clustering-hierarchical-samples
+  name: Hierarchical clustering
+  data_name: 'Hierarchical clustering of samples'
+  version: 0.0.1
+  type: data:sample:clustering:hierarchical
+  category: analyses
+  persistence: TEMP
+  description: >
+    Hierarchical clustering of samples.
+  input:
+    - name: exps
+      label: Expression
+      type: list:data:expression
+    - name: sampleids
```

Comment:
You should be able to get that from expressions `{% for e in exps %}{{e|sample_id}} {% endfor %}`. This also works:
```
{% for e in exps %}{{e|sample_slug}} {% endfor %}  # a list of sample slugs
{% for e in exps %}{{e|sample_name}} {% endfor %}  # a list of sample names
```